getEvent/index.ts

```diff
@@ -0,0 +1,32 @@
+import { AzureFunction, Context, HttpRequest } from '@azure/functions';
+import getConnection from '../shared/services/db';
+import { getEventSchema } from '../shared/event/event.schema';
+import FunctionError from '../shared/services/error';
+import EventModel from '../shared/event/event.model';
+
+const getResObject = (statusCode: Number | undefined, responseMessage: any) => {
+    return { status: statusCode ?? process.env.SERVER_ERROR_CODE, body: responseMessage };
+};
+
+const getEvent: AzureFunction = async (context: Context, req: HttpRequest): Promise<void> => {
+    try {
+        await getConnection();
+
+        const { error } = getEventSchema.validate(req);
+        if (error) {
+            throw new FunctionError(parseInt(process.env.VALIDATION_ERROR_CODE, 10), error.message);
```

Comment:
```suggestion
           context.res = getResObject(400, error.message)
            context.done();
```

There is no need to throw an error in a try-catch section, you can send the res directly.
I write this here but you should change it in the other files as well.